cdk/lib/__snapshots__/security-hq.test.ts.snap

```diff
@@ -202,6 +166,9 @@ Object {
           "Ref": "AMISecurityhq",
         },
         "InstanceType": "t4g.large",
+        "MetadataOptions": Object {
```

Comment:
Included in [v38.0.0](https://github.com/guardian/cdk/releases/tag/v38.0.0), this _should_ be OK.